<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoubleRangeSliderPropertyWidget</class>
 <widget class="QWidget" name="DoubleRangeSliderPropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
   <item row="1" column="0">
    <widget class="pqLineEdit" name="ThresholdBetween_0">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ctkDoubleRangeSlider" name="RangeSlider" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="pqLineEdit" name="ThresholdBetween_1">
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QPushButton" name="placeHolder">
     <property name="text">
      <string>..</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDoubleRangeSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleRangeSlider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqLineEdit</class>
   <extends>QLineEdit</extends>
   <header>pqLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
